.github/workflows/_build-img-nucliadb.yml

```diff
@@ -9,6 +9,11 @@ on:
         type: string
         required: false
         default: ""
+      image-version:
+        description: "Image version."
+        type: string
+        required: true
+        default: ""
       context:
```

Comment:
Same for this parameter and some of the others (platforms, cache mode, skip. No longer used I think

Reply:
I removed the platform, cache mode, and skip. Let's keep context, as it could be helpful to and is supported by the new action.